code/game/objects/breakable.dm

```diff
@@ -25,11 +25,6 @@
 	var/damaged_sound	//Audible sound when the object is damaged by an attack, but not fully broken. Defaults to glanced_sound if unset.
 	var/glanced_sound	//Audible sound when the object recives a glancing attack not strong enough to damage it.
 
-/obj/New()
-	..()
-	if(breakable_flags)
-		breakable_init()
```

Comment:
?

Reply:
I moved that to objs.dm since it has more than just `breakable_init()` now.